pkg/webconsolevalidation/server.go

```diff
@@ -20,19 +20,28 @@ import (
 // K8sClient is used to get the webconsolerequest resource from UUID and namespace.
 var K8sClient ctrlclient.Client
 
+// Function variables to allow for mocking in tests.
+var (
```

Comment:
Might I ask why you do not just send this into the `InitServer` function as arguments?

Reply:
Updated as suggested and also removed the other pkg-scoped var `K8sClient` previously defined in this file.